main/src/ca/uwaterloo/flix/language/debug/FormatType.scala

```diff
@@ -251,7 +251,12 @@ object FormatType {
         fields.map(visit(_, Mode.Type)).mkString("(", ", ", ")")
     }
 
-    visit(tpe00, Mode.Type)
+    // TODO: Remove after we're confident in the formatter.
```

Comment:
A good test is to run `--documentor` since that will format all top-level signatures.

Reply:
No errors in documentation.